editions/tw5.com/tiddlers/concepts/Tiddler Colour Cascade.tid

```diff
@@ -6,11 +6,13 @@ type: text/vnd.tiddlywiki
 
 The tiddler colour cascade is a [[cascade|Cascades]] used to choose which colour should be used for a particular tiddler.
 
+Core tiddler colour cascades can be found in $:/core/macros/tag , $:/core/ui/Components/tag-link , $:/core/ui/EditTemplate/tags , $:/core/ui/TagPickerTagTemplate , $:/core/ui/TagTemplate and $:/core/ui/ViewTemplate/title
```

Comment:
I don't think it's necessary to have a space before each comma.

Reply:
You're right. I've posted a new PR. Please close this one. Thanks.

Reply:
Thanks @twMat